api/graphql/reservations/reservation_types.py

```diff
@@ -327,7 +327,9 @@ def resolve_cancel_details(self, info: ResolveInfo) -> Optional[str]:
     def resolve_order_uuid(self, info: ResolveInfo) -> Optional[str]:
         payment_order = self.payment_order.first()
         return (
-            payment_order.order_id if payment_order and payment_order.order_id else None
+            payment_order.remote_id
```

Comment:
I'd suggest (not strongly) to refactor this to;
```
uuid = getattr(payment_order, "remote_id", None)
return uuid
```
For readability

Reply:
Fixed! 👍